hack/postgres-passwords-migration.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/bash
+# This script must be run to migrate all database passwords
+# before upgrading the synopsys-operator
+
+function usage(){
+  echo -e "usage:        ${0} [kubectl/oc] [NAMESPACE1] [NAMESPACE2] [...]\n"
+  echo -e "kubectl/oc:   Type which client you would like to use\n"
+  echo -e "NAMESPACE:    The blackduck namespace to migrate the secret from\n"
+}
+
+# check for number of arguments
+[ "$#" -lt 2 ] && echo -e "ERROR: At least one NAMESPACE must be provided.\n" && usage && exit
+
+# get the client; kubectl or oc
+client=$1
+echo -e "Using $client as the client, please make sure you have it on the machine you're running this script on\n"
+# move the argument pointer past the first item to process namespaces
+shift
+
+# Get ADMIN_PASSWORD, USER_PASSWORD, POSTGRES_PASSWORD from synopsys-operator blackduck-secret
+ADMIN_PASSWORD=$($client get secret -n synopsys-operator blackduck-secret -o jsonpath='{.data.ADMIN_PASSWORD}')
+# USER_PASSWORD
+USER_PASSWORD=$($client get secret -n synopsys-operator blackduck-secret -o jsonpath='{.data.USER_PASSWORD}')
+# POSTGRES_PASSWORD
+POSTGRES_PASSWORD=$($client get secret -n synopsys-operator blackduck-secret -o jsonpath='{.data.POSTGRES_PASSWORD}')
+
+for namespace
```

Comment:
namespace hasn't been set to anything here.   You intend to iterate over the remaining args as namespaces it seems.

Reply:
Yup, that's what I've also put in the usage.

Reply:
Whoops, mis-read this line.  Disregard my comment.